### PR TITLE
Hostname is added for service info

### PR DESCRIPTION
**Pull request by melanietom**
For bgpinspect we need host name to ip translation hence all netplugins register service info with their host name included.

### CODE REVIEW

File: consulService.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type consulServiceState struct {
 	Port        int           // Port number where its listening
 	SessionID   string        // session id assigned by consul
 	stopChan    chan struct{} // Channel to stop ttl refresh
+	Hostname    string        // Host name where its running
```
Comment (shaleman) [inline consulService.go]: Looks like this field is not being set. Can you please set it in RegisterService()?

File: etcdService.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ type etcdServiceState struct {
 	TTL         time.Duration // TTL for the service
 	HostAddr    string        // Host name or IP address where its running
 	Port        int           // Port number where its listening
+	Hostname    string        // Host name where its running
```
Comment (shaleman) [inline etcdService.go]: same comment as above..

**Comment by shaleman** [comment]
LGTM other than the minor comment..

File: consulService.go (hunk 1 above)
Comment (melanietom) [inline consulService.go]: Done

File: etcdService.go (hunk 2 above)
Comment (melanietom) [inline etcdService.go]: Done

**Pull request merged**